### PR TITLE
fix(images): fix broken images on the Environments page

### PR DESCRIPTION
**Pull request by jstrachan**
switch to using the standard stack icons

### CODE REVIEW

**Comment by jstrachan** [comment]
note that to be able to work locally we can't serve images from `/_assets/icon/` due to this line:
https://github.com/jstrachan/fabric8-ui/blob/3b21b75f2f634a0e240f7b32c24467ead79080ab/config/webpack.common.js#L292

I've no clue how to fix that so I've coped the `assets/icon/icon-*.png` files to `assets/images` for now. 

We could update the CSS to refer to the icons at this new location of `/assets/images/` and remove the duplicates?

**Comment by joshuawilson** [comment]
@mindreeper2420 Can you take a look at this.

**Comment by fabric8cd** [comment]
@jstrachan fabric8/fabric8-ui:SNAPSHOT-PR-2010-1 fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2010-fabric8-ui.badger.fabric8.io

**Comment by jstrachan** [comment]
as per the chat on mattermost, I've now moved the stack/pipeline icons from `assets/icon` to `assets/images`

**Comment by fabric8cd** [comment]
@jstrachan fabric8/fabric8-ui:SNAPSHOT-PR-2010-2 fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2010-fabric8-ui.badger.fabric8.io

**Comment by fabric8cd** [comment]
@jstrachan fabric8/fabric8-ui:SNAPSHOT-PR-2010-3 fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2010-fabric8-ui.badger.fabric8.io

File: src/a-runtime-console/kubernetes/model/kubernetesresource.model.ts [hunk 1]
```diff
@@ -61,12 +61,47 @@ export class KubernetesResource implements BaseEntity {
...
+      this.icon = "/_assets/images/icon-stack-wildfly.png";
+    }
     // TODO any other annotations we should look for?
     this.description = this.annotations['description'] || templateAnnotations['description'] || '';
 
     this.openShiftConsoleUrl = openShiftBrowseResourceUrl(this, currentOAuthConfig());
   }
 
+  isFabric8Icon(url, name) {
+    if (url && (url.startsWith("img/icons/") || url.startsWith("/img/icons/"))) {
```
Comment (joshuawilson) [inline src/a-runtime-console/kubernetes/model/kubernetesresource.model.ts]: I thought the idea was to point to the `/assets/images` directory?

Comment (jstrachan) [reply src/a-runtime-console/kubernetes/model/kubernetesresource.model.ts]: that is the idea. 

This code is about transforming **old** fabric8 icon URLs generated via the fabric8-maven-plugin and annotated onto released `Deployment` resources to URLs which work nicely on the OSIO console. i.e. the code replaces `img/icons/spring.png -> /_assets/images/icon-stack-spring.png`

**Pull request merged**